odoo/models.py

```diff
@@ -2643,8 +2658,8 @@ def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=Fals
         :param list groupby: list of groupby descriptions by which the records will be grouped.
                 A groupby description is either a field (then it will be grouped by that field)
                 or a string 'field:granularity'. Right now, the only supported granularities
-                are 'day', 'week', 'month', 'quarter' or 'year', and they only make sense for
-                date/datetime fields.
+                are 'day', 'week', 'month', 'quarter' or 'year'; as well as 'relative_month',
```

Comment:
`'hour'` is also supported, and is currently not documented.
And the grammar of the sentence is not correct: your sentence says that `relative_XXX` only make sense for date/datetime fields, but not for the former ones.